api/src/test/java/org/apache/iceberg/metrics/TestDefaultTimer.java

```diff
@@ -101,14 +103,7 @@ public void closeableTimer() throws InterruptedException {
   @Test
   public void measureRunnable() {
     Timer timer = new DefaultTimer(TimeUnit.NANOSECONDS);
-    Runnable runnable =
```

Comment:
the goal here is actually to measure the duration of the `runnable`, so this isn't something that we want to replace with Awaitility